scripts/dxf2wkt.py

```diff
@@ -516,9 +550,23 @@ def parse_dxf_file(
             except Exception as e:
                 coords = f"Could not extract coordinates: {e}"
 
-            logging.warning(
-                f"Skipped {entity.dxftype()} at ({coords[0]:.2f}, {coords[1]:.2f})"
-            )
+            # Fix the logging warning to handle string coords properly
```

Comment:
I don't understand this comment, can you make it more clear?

Reply:
This is indeed vague. I just wanted to safely logging stuff. Change in the new commit!